frontend/pages/hosts/details/modals/VitalsModal/VitalsModal.tests.tsx

```
@@ -0,0 +1,374 @@
+    expect(screen.getAllByText("True").length).toBeGreaterThan(0);
+    expect(screen.getAllByText("False").length).toBeGreaterThan(0);
+  });
+
+  it("treats a negative battery level as unknown, since Apple reports -1 when it can't determine one", () => {
+    const host = buildFullyPopulatedHost({ battery_level: -1 });
+
+    const { container } = render(
+      <VitalsModal host={host} vitalsData={host} mdm={host.mdm} onExit={noop} />
+    );
+
+    expect(findValueCell(container, "Battery level")?.textContent).toBe(DEFAULT_EMPTY_CELL_VALUE);
```

ESLint

[error] Replace `DEFAULT_EMPTY_CELL_VALUE` with `⏎······DEFAULT_EMPTY_CELL_VALUE⏎····`

(prettier/prettier)